### ISSUE TITLE
[BUG] "xtdc run" does not display the output of the console application on macOS

### ISSUE THREAD

**Issue by jimorc** [gammasoft71/xtd#185]
# Describe the bug

I get the following when executing either
xtdc build
or xtdc run
on the code shown in the Writing console application page at gammasoft71.wixsite.com:

2022-08-18 09:51:22.573 xcodebuild[10049:3636229] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2022-08-18 09:51:22.574 xcodebuild[10049:3636229] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore

# To Reproduce

1. Create a test program:
```
#include <xtd/xtd>

using namespace xtd;

class console1{
    public:
        console1() {}
};

int main() {
    console1 {};
    console::write_line("Hello, World!");
}
```

2. Create CMakeLists.txt:
```
cmake_minimum_required(VERSION 3.3)

project(console1)
find_package(xtd REQUIRED)

add_sources(console1.cpp)
target_type(CONSOLE_APPLICATION)
```

3. Execute xtdc build
or xtdc run

# Expected behaviour

Program should build or build and execute

# Screenshots

N/A

# Desktop (please complete the following information)

 - OS: MacOS 12.5
 - Chip: M1 Pro
- xtd version: From master branch downloaded yesterday

**Additional context**

N/A

**Workaround**

None known

**Comment by gammasoft71** [comment]
# Bug

Indeed, the display of the stdout of the application is not displayed.
The program has been compiled and executed but the output is not displayed.

# Workaround

While waiting for the correction:

```shell
xtdc build
./build/Debug/console1
```

# Remarks

Only on macOS.

**Comment by gammasoft71** [comment]
Fix in commit [753717d](https://github.com/gammasoft71/xtd/commit/753717d1d1e6529adc140f253769af6927429001)
